docs/ig.md

```diff
@@ -222,3 +222,40 @@ List of flags:
   `--auto-mount-filesystems` to ig to automatically mount those filesystems.
 - `--pid=host` runs in the host PID namespace. Optional on Linux. This is necessary on Docker Desktop on Windows because
   /host/proc does not give access to the host processes.
+
+### Using ig in a Kubernetes pod
```

Comment:
I'm wondering if we can add more information on this case. We should describe what is the motivation to run `ig` and not `ig-k8s`. (I suppose it's because by using ig we can get the output directly without needing to execute kubectl-gadget, right?)